helm/v1beta1/types_helm_chart_repository.go

```diff
@@ -83,6 +83,13 @@ type ConnectionConfig struct {
 	// The namespace for this secret is openshift-config.
 	// +optional
 	TLSClientConfig configv1.SecretNameReference `json:"tlsClientConfig,omitempty"`
+
+	// PgpKeys is an optional reference to a list of config maps by name containing the public pgp keys of signed charts.
```

Comment:
What happens if I don't provide anything here?

Reply:
The case is very well valid for charts which are not signed. If the repository does not contain any signed charts then this field will be ignored.

Reply:
We are currently working on https://issues.redhat.com/browse/HELM-410(chart verifier cli tool) to run signature check for a chart. If the PGPKey is not specified and the chart is not signed we will be mentioning the check as passed. Need to create a rest api to be consumed by console , but the logic as part of https://issues.redhat.com/browse/HELM-410 remains intact.

Reply:
> The case is very well valid for charts which are not signed. If the repository does not contain any signed charts then this field will be ignored.

I would suggest adding this context to the description so that generated docs include this context